d2game/d2gamescreen/game.go

```diff
@@ -92,7 +92,7 @@ func CreateGame(
 		ticksSinceLevelCheck: 0,
 		mapRenderer: d2maprenderer.CreateMapRenderer(asset, renderer,
 			gameClient.MapEngine, term, startX, startY),
-		escapeMenu:    d2player.NewEscapeMenu(navigator, renderer, audioProvider, guiManager, asset),
+		escapeMenu:    d2player.NewEscapeMenu(navigator, renderer, audioProvider, guiManager, asset, gameClient),
```

Comment:
Why are you giving the gameClient to the escape menu? You could also get away with just giving it the player, as this is all that you need.

Reply:
Hey @juander-ux!
Thanks for taking a look at it.
Yes, only giving the player class would be a good idea. I will change that.